packages/shared/src/dto/workflows/create-workflow-dto.ts

```diff
@@ -2,8 +2,6 @@ import { PreferencesRequestDto, StepCreateDto, WorkflowCommonsFields } from './w
 import { WorkflowCreationSourceEnum } from '../../types';
 
 export type CreateWorkflowDto = WorkflowCommonsFields & {
-  workflowId: string;
```

Comment:
was added to FrameworkCommonsFields

Reply:
why? it's not a part of the create no? it's generatred on the backend, what it has to do with this PR